RichText-web-module/src/main/webapp/resources/celRteJS/tinyMCE4/loadTinyMCE-async.js

```diff
@@ -89,6 +89,9 @@
     try {
       const editor = event.target;
       console.debug('celFinishTinyMCEStart: start', event);
+      const origElemClassNames = editor.getElement().className.split(' ')
+        .filter(cssClass => cssClass.startsWith('celEditorBody_'));
+      editor.getBody().className  += ' ' + origElemClassNames.join(' ');
```

Comment:
use `classList` instead of `split`/`join`
```suggestion
      const origElemClassNames = [...editor.getElement().classList]
        .filter(cssClass => cssClass.startsWith('celEditorBody_'));
      editor.getBody().classList.add(...origElemClassNames);
```